app/views/admin/roles/_form.html.erb

```diff
@@ -5,48 +5,116 @@
+          {
+            text: whip_organisation&.label,
+            value: whip_organisation&.id,
+            selected: whip_organisation&.id == role.whip_organisation_id
+          }
+        end
+    } %>
 
-  <fieldset>
-    <%= form.text_area :responsibilities, rows: 20, class: "previewable", data: {
-      module: "paste-html-to-govspeak"
+    <%= render "govuk_publishing_components/components/select", {
+      id: "role_role_payment_type_id",
+      label: "Payment options",
+      name: "role[role_payment_type_id]",
+      options: ([nil] + RolePaymentType.all).map do |role_payment_type|
+        {
+          text: role_payment_type&.name,
+          value: role_payment_type&.id
+        }
+      end
     } %>
-  </fieldset>
-  <p class="warning">
-    <% if show_instantly_live_warning %>
-      Warning: changes to roles appear instantly on the live site.
-    <% end %>
-    <% if show_consult_gds_warning %>
-      Do not create ministerial roles without consulting GDS.
-    <% end %>
-  </p>
-  <%= form.save_or_continue_or_cancel cancel: admin_roles_path %>
+
+    <%= render "govuk_publishing_components/components/select", {
+      id: "role_attends_cabinet_type_id",
+      label: "Attends cabinet options",
+      name: "role[attends_cabinet_type_id]",
+      options: ([nil] + RoleAttendsCabinetType.all).map do |role_attends_cabinet_type|
+        {
+          text: role_attends_cabinet_type&.name,
+          value: role_attends_cabinet_type&.id
+        }
+      end
+    } %>
+
+    <%= render "components/autocomplete", {
+      id: "role_worldwide_organisation_ids",
+      label: {text: "Worldwide organisations"},
+      name: "role[worldwide_organisation_ids][]",
+      select: {
+        options:  options_from_collection_for_select(WorldwideOrganisation.all, 'id', 'name', role.worldwide_organisation_ids),
+        multiple: true
+      }
+    } %>
+
+    <%= render "components/govspeak-editor", {
+      label: {
+        text: "Responsibilities"
+      },
+      id: "role_responsibilities",
+      name: "role[responsibilities]",
+      value: role.responsibilities_without_markup,
+      rows: 20
+    } %>
+
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Changes to roles appear instantly on the live site."
+    } if show_instantly_live_warning %>
+
+    <%= render "govuk_publishing_components/components/warning_text", {
+      text: "Do not create ministerial roles without consulting GDS."
+    } if show_consult_gds_warning %>
+
+    <div class="govuk-button-group">
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Save",
+        name: "save",
+        value: "Save",
+        type: "submit"
+      } %>
+      <%= render "govuk_publishing_components/components/button", {
+        text: "Save and continue",
+        name: "save_and_continue",
+        value: "Save and continue",
+        type: "submit"
+      } %>
```

Comment:
This is usually styled as secondary

<img width="704" alt="Screenshot 2023-03-28 at 11 04 47 am" src="https://user-images.githubusercontent.com/4599889/228202531-6548b3e5-7c29-44f3-aa99-4d72489180cd.png">

Reply:
👍